### ISSUE TITLE
(v1.5.1) Invalid prop: type check failed for prop "src". Expected String, Object, got Promise

### ISSUE THREAD

**Issue by nicholaszuccarelli** [arkokoley/pdfvuer#53]
After upgrading to 1.5.1, I am getting the following console issue:

> Invalid prop: type check failed for prop "src". Expected String, Object, got Promise

The src value is certainly a string that I am passing in. The PDF still does load, but this error is thrown every time I try to load a file.

**Comment by ahoiroman** [comment]
Got the same problem. I can't even check using something like 

```
<pdf :src="pdfdata" ... v-if="!isPromise(pdfdata)">

...

isPromise (p){
				return p && Object.prototype.toString.call(p) === "[object Promise]";
			}
```

because the check initially fails and the component is not rendered again.

**Comment by Roywcm** [comment]
I have the same issue.

**Comment by ThomasVanBroekhoven** [comment]
I'm also having this issue.

**Comment by nicholaszuccarelli** [comment]
Any updates?

**Comment by beltranrengifo** [comment]
Same here!
![Captura de pantalla 2020-04-17 a las 14 07 06](https://user-images.githubusercontent.com/19485114/79567564-c07b5c00-80b4-11ea-8a0f-843e49c87c27.png)

**Comment by arkokoley** [comment]
Fixed this in v1.6.0

**Comment by beltranrengifo** [comment]
Thank you!
